pkg/edgeview/src/copyfile.go

```diff
@@ -64,6 +74,18 @@ func runCopy(opt string) {
 		return
 	}
 
+	ok := checkBlockedDirs(file)
```

Comment:
How does this prevent a tar of e.g., /persist picking up /persist/vault?
Isn't the check needed inside somewhere?

Reply:
the file is actually the whole path here, with the /persist/vault/ as the prefix. so it will be excluded.